src/System.Reflection.DispatchProxy/src/System/Reflection/DispatchProxy.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Hmm... the analyzer that verifies license headers did not kick-off. I need to investigate.

Reply:
Yeah I noticed that as well. Might be a regression in the current 9.0 SDK. The 8.0.100 SDK worked for me.

Reply:
I figured it out - I forgot to add the projects to the sln, so they were not even building.